test/functional/feature_includeconf.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+# Copyright (c) 2018 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Tests the includeconf argument
+
+Verify that:
+
+1. adding includeconf to the configuration file causes the includeconf
+   file to be loaded in the correct order.
+2. includeconf cannot be used as a command line argument.
+3. includeconf cannot be used recursively (ie includeconf can only
+   be used from the base config file).
+4. multiple includeconf arguments can be specified in the main config
+   file.
+"""
+import os
+
+from test_framework.test_framework import BitcoinTestFramework, assert_equal
+import tempfile
+
+class IncludeConfTest(BitcoinTestFramework):
+    def setup_chain(self):
```

Comment:
micronit: all other tests have `set_test_params()` as the first method. I think that makes more logical sense since that method is called by `__init__()`. Do you mind doing the same here?